lib/inc/facter/execution/execution.hpp

```diff
@@ -11,10 +11,16 @@
 #include <utility>
 #include <stdexcept>
 #include <functional>
-#include "../util/option_set.hpp"
-#include "../util/environment.hpp"
 #include "../export.h"
 
+#include <leatherman/util/environment.hpp>
+#include <leatherman/util/option_set.hpp>
+#include <leatherman/execution/execution.hpp>
+#include <leatherman/execution/execution.hpp>
+
+namespace lth_util = leatherman::util;
+namespace lth_exec = leatherman::execution;
+
 namespace facter { namespace execution {
 
     /**
```

Comment:
I hadn't really thought about it.  Could perhaps remove them from facter's public API and just make users use leatherman?  No one uses this stuff yet, so now would be the time to do that.